lua/class_definitions.lua

```diff
@@ -407,7 +407,7 @@ classes = {
             { name = "cough", rval = nil, args = { "bool", "int" } },
             { name = "cough", rval = nil, args = { } },
             { name = "craft", rval = nil, args = { } },
-            { name = "crafting_allowed", rval = "bool", args = { } },
+            { name = "crafting_allowed", rval = "bool", args = { "string" } },
```

Comment:
I don't think any existing lua code uses this function. I considered removing it, but decided to make it work.